logging/conf/agent.yml

```diff
@@ -0,0 +1,76 @@
+integrations:
+  prometheus_remote_write:
+  - basic_auth:
+      password: ${PROM_PASS}
+      username: ${PROM_USER}
+    url: <your_prom_url>
+  agent:
+    enabled: true
+    relabel_configs:
+    - action: replace
+      source_labels:
+      - agent_hostname
+      target_label: instance
+    - action: replace
+      target_label: job
+      replacement: "integrations/agent-check"
+    metric_relabel_configs:
+    - action: keep
+      regex: (prometheus_target_.*|prometheus_sd_discovered_targets|agent_build.*|agent_wal_samples_appended_total|process_start_time_seconds)
+      source_labels:
+      - __name__
+  # Add here any snippet that belongs to the `integrations` section.
+  # For a correct indentation, paste snippets copied from Grafana Cloud at the beginning of the line.
```

Comment:
this section of comments seems unnecessary

Reply:
removed